.github/.coderabbit.yml

```
@@ -1,4 +1,4 @@
-language: "typescript"
+language: "ja-JP"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)